client/src/components/Admin/OrganizationEdit/ContactDetails.js

```diff
@@ -1,6 +1,59 @@
-import { Grid, TextField } from "@mui/material";
+import { Grid } from "@mui/material";
 import { TabPanel } from "components/Admin/ui/TabPanel";
+import { InputAdornment, TextField } from "@mui/material";
 import Label from "../ui/Label";
+import { styled } from "@mui/system";
+
+const CustomTextField = styled(TextField)({
+  "& .MuiOutlinedInput-root": {
+    borderRadius: "4px",
+    backgroundColor: "white",
+    paddingLeft: "0",
+    "& fieldset": {
+      borderColor: "grey",
+    },
+    "& input::placeholder": {
+      color: "#B0BEC5",
+      opacity: 1,
+    },
+  },
+  "& .MuiInputAdornment-root": {
+    backgroundColor: "#F4F6F8",
+    padding: "22px 8px 22px 8px",
+    borderRight: "1px solid grey",
+    borderRadius: "4px 0 0 4px",
+    color: "#B0BEC5",
+    margin: "-20px 0",
+  },
+});
```

Comment:
since we're not reusing this I would just inline the styles in a `<TextField />`
```const CustomTextFieldComponent = ({
  id,
  name,
  placeholder,
  value,
  onChange,
  onBlur,
  touched,
  errors,
  startAdornment,
}) => (
  <TextField
    id={id}
    sx={{
      "& .MuiOutlinedInput-root": {
        borderRadius: "4px",
        backgroundColor: "white",
        paddingLeft: "0",
        "& fieldset": {
          borderColor: "grey",
        },
        "& input::placeholder": {
          color: "#B0BEC5",
          opacity: 1,
        },
      },
      "& .MuiInputAdornment-root": {
        backgroundColor: "#F4F6F8",
        padding: "22px 8px 22px 8px",
        borderRight: "1px solid grey",
        borderRadius: "4px 0 0 4px",
        color: "#B0BEC5",
        margin: "-20px 0",
      },
    }}
    name={name}
    placeholder={placeholder}
    value={value}
    onChange={onChange}
    onBlur={onBlur}
    helperText={touched ? errors : ""}
    error={Boolean(touched && errors)}
    InputProps={{
      startAdornment: startAdornment && (
        <InputAdornment position="start">{startAdornment}</InputAdornment>
      ),
    }}
  />
);
```

Reply:
What do you mean by this? I thought by default it is using that style? If i use inline style like that I think i would have to include that inline style in every textfield ?

Reply:
okay, fixed both to check if theres a handle and the inline styles like you said above. Let me know if you need any changes to to.